source/user-manual/reference/ossec-conf/wodle-command.rst

```diff
@@ -1,5 +1,8 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
+.. meta::
+  :description: Learn more about the Wazuh local configuration, including configuration sections and supported installations, in this section of our documentation. 
```

Comment:
I'd rewrite this meta since the content is not consistent with the topic of this section.